src/interfaces/store-interfaces/state-interfaces/continuation-in-state-interface.ts

```diff
@@ -1,4 +1,6 @@
 // TODO: Add properties as needed
 export interface ContinuationInStateIF {
-  existingBusinessValid: boolean
+  step1Valid: boolean
+  step2Valid: boolean
```

Comment:
I might rename these in a future ticket, I dunno.

The alternative is `yourBusinessInHomeJurisdictionValid` :/

Or maybe just `businessHomeValid` (which is consistent with the route and view names.

Reply:
Never mind, I just changed it.